src/renderers/shared/server/ReactDOMNodeStreamRenderer.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactDOMNodeStreamRenderer
+ */
+
+'use strict';
+
+var invariant = require('fbjs/lib/invariant');
+var React = require('react');
+var ReactDOMPartialRenderer = require('ReactDOMPartialRenderer');
+
+var Readable = require('stream').Readable;
```

Comment:
Is there any where we can read someone arguing _against_ depending on the `readable-stream` package? This would make it easier to evaluate the counter-arguments for someone not too involved in the details. Seems non-obvious to me.